.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 	parser: 'babel-eslint',
 	env: {
 		browser: true,
+		jest: true,
 		mocha: true,
```

Comment:
Once we merge this PR and #14364. We also need to run [jest-codemods](https://github.com/skovhus/jest-codemods) to replace:
* `after` => `afterAll
* `before` => `beforeAll`
* `context` => `describe`
Then we can remove it completely together with Mocha dependency and code which powers the actual test runner :)